woocommerce/google-listings-and-ads
# WP 6.4 Compatibility: Fix the inappropriate Popover width

### Changes proposed in this Pull Request:

Closes #2204

- Fix the inappropriate Popover width due to [a breaking change in WordPress 6.4.0](https://github.com/WordPress/gutenberg/blob/wp/6.4/packages/components/CHANGELOG.md#breaking-changes).
   - > Make the `Popover.Slot` optional and render popovers at the bottom of the document's body by default.
- Remove a long unused scss file.
   - *js/src/product-feed/product-statistics/product-status-help-popover/index.scss*
   - This file had been removed in [#517](https://github.com/woocommerce/google-listings-and-ads/pull/517) but was accidentally added back in the later merged [#512](https://github.com/woocommerce/google-listings-and-ads/pull/512) due to file renaming and incorrect `trunk` sync.

### Screenshots:

![image](https://github.com/woocommerce/google-listings-and-ads/assets/17420811/5bbd871c-5b31-4b9d-9ee5-7d6c63569620)

### Detailed test instructions:

1. Set up a WordPress version < 6.4.0
2. Go to the Product Feed page
3. Click the question icon in the Overview title to show the popover
4. View the originally expected width of the popover
   - If testing with the `define( 'SCRIPT_DEBUG', true );` flag or `npm run start:hot`, You may see the following warning in the DevTool Console.
      ![1](https://github.com/woocommerce/google-listings-and-ads/assets/17420811/b1093f22-29da-4af7-913b-4a083e8fb3b6)
   - This is because `Popover` didn't have the `inline` prop before WP 6.4. The warning is a reminder to devs to make sure the prop is not misplaced or misused, and will not cause any problems in the actual result.
5. Set up a WordPress version >= 6.4.0
6. Refresh the current webpage
7. Show the same popover and check if the width is appropriate

### Changelog entry

> Fix - WordPress 6.4 Compatibility: Set an appropriate width for the content in the Popover component.
